data/human/human missions.txt

```diff
@@ -7411,3 +7411,1825 @@ mission "Adelita 5"
+			`	Wynnafrith says a few quiet words into an intercom, and a door behind her opens. "The Governor is ready for you."`
+			choice
+				`	(Enter the room.)`
+
+			`	You enter a large richly furnished room. At the center of the room is an imposing desk, with a slightly graying man seated behind it. There is something immediately familiar about his face.`
+			`	"Ah, Captain <last>. At last we can meet face to face, and I can thank you properly." The Governor stands. "You may not remember me, it has been some years since you last laid eyes on me, and," the governor gestures to himself, "I looked quite different back then."`
+			`	"I was a stowaway on your ship. You found me in a food crate, and you took me here. I'm Timothy Radrickson, Governor Timothy Radrickson now."`
+			`	Timothy sits back down and indicates that you should do the same. "It took me some time. But through some good decisions, and quite a bit of luck, I've been able to claw my way to a level of respectability and wealth. I've really been able to make something of myself."`
+			`	His face opens up into a large grin. "Then again, compared to what you've done in all this time, my successes hardly register. Still, as meager as they may be, I owe it all to you and your kind gesture. Which is why I've invited you here, to thank you and pay you back properly."`
+			`	Timothy taps a display on his desk. "First, I am trasnfering 125,000 credits to your account to pay you back for the crate of food, the fuel, and the additional expenses of taking care of me on your ship; plus interest, of course."`
+			choice
+				`	(Accept the money.)`
+					goto moneyaccept
+				`	(Refuse the money.)`
+
+			`	You begin to refuse the generous offer, but Timothy holds up a hand to stop you. "I am doing quite well, and this discharges a debt that has been weighing on me. Besides, the money's already been sent!"`
+
+			label moneyaccept
+			`	Timothy pulls out a bottle beneath his desk and two glass cups. "To celebrate our reunion."`
+			`	He pours the brown liquid in the cups and pushes one to you. "Things have gone well for me, and are only going to get better from here on out."`
+			`	He takes a sip and says, "We've just launched our largest freighter fleet to date." He sits back in his desk, relaxed, and gets a dreamy look in his eyes. "When it returns from Wayfarer, the government of Rand will finally have enough credits to make some real changes around here. Give something back to the working man."`
+			choice
+				`	(Congratulate Timothy.)`
+				`	(Take a sip from your glass.)`
+				`	(Complain about the living standards of workers on Rand.)`
+				`	(Complain about hand-outs for the poor of Rand.)`
+
+			`	His eyes refocus on you. "You'd be proud, <first>. I am going to make a real difference here..."`
```

Comment:
I just thought his response would be the same regardless of what you said. 

If say nothing, or give him a congrats, he says you'd be proud because he's done good work.

If you complain about something, he's also going to say you'd be proud because he's making changes.